Sample Applications/DataBindingDemo/App.cs

```diff
@@ -14,6 +16,8 @@ public partial class App : Application
     {
         public User CurrentUser { get; set; }
         public ObservableCollection<AuctionItem> AuctionItems { get; set; } = new ObservableCollection<AuctionItem>();
+        private Grid FocusedGrid { get => focusedGrid; set => focusedGrid = value; }
+        private Grid focusedGrid;
```

Comment:
Just use an automatic property here, no reason to have a backing field, especially since it's a private member.

Reply:
removed